docs/installation/java-installation.rst

```diff
@@ -1,32 +1,26 @@
 .. _java-installation-label:
 
-=================
-Installing Java 8
-=================
+===============
+Installing Java
+===============
 Prerequisites: :ref:`presto-admin-installation-label` and :ref:`presto-admin-configuration-label`
 
-The Oracle Java 1.8 JRE (64-bit), update 45 or higher, is a prerequisite for Presto. If a suitable 64-bit version of Oracle Java 8 is already installed on the cluster, you can skip this step.
+JDK 11, is a prerequisite for Presto. If a suitable 64-bit version of JDK 11 is already installed on the cluster, you can skip this step.
```

Comment:
A Java 11 runtime is a requirement for Presto.

Reply:
Maybe should link to https://prestosql.io/docs/current/installation/deployment.html#requirements